src/GlobalInstaller.php

```diff
@@ -210,6 +210,6 @@ private function supportsGlobalPath(PackageInterface $package): bool
             return false;
         }
 
-        return !in_array($package->getPrettyName(), $this->options->excluded ?? [], true);
+        return !in_array($package->getPrettyName(), $this->options->exclude ?? [], true);
```

Comment:
Could you unit test this as well? 

Only need to provide a small config which excludes that name, and assert that the path is in the vendor folder instead of the global verdor.